universal/algos/rprt.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+from ..algo import Algo
+from .. import tools
+import numpy as np
+import pandas as pd
+
+
+class RPRT(Algo):
+    """ Reweighted Price Relative Tracking System for Automatic Portfolio Optimization
+        Reference:
+            Zhao-Rong Lai and Pei-Yi Yang and Liangda Fang and Xiaotian Wu.
+            Reweighted Price Relative Tracking System for Automatic Portfolio Optimization, 2018.
+            https://ieeexplore.ieee.org/document/8411138
+    """
+
+    PRICE_TYPE = 'raw'
+    REPLACE_MISSING = True
+
+    def __init__(self, window=5, eps=50, theta=0.8, **kwargs):
+        """
+        :param window: Lookback window.
+        :param eps:
+        :param theta:
+        """
+
+        super(RPRT, self).__init__(min_history = window, **kwargs)
```

Comment:
You could simplify this to `super().__init__(min_history = window, **kwargs)` (using args is a legacy thing from python 2)

Reply:
Didn't know they are not needed, thx !